docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,192 @@
 
-1.  **preimage requests**: The rollup can request arbitrary data of at
-    most 4kBytes, provided that it knows its (blake2b) hash. The request
-    is fulfilled by the rollup node (see [Populating the Reveal Channel](#populating-the-reveal-channel)).
-2.  **metadata requests**: The rollup can request information from the
-    protocol, namely the address and the origination level of the rollup
-    node itself. The rollup node retrieves this information through RPCs
-    to answer the rollup.
+- A rollup node can request an arbitrary data page up to 4KB at a time if it knows the blake2b hash of the page, known as _preimage requests_.
+To transfer more than 4KB of data, rollups must use multiple pages, such as a single file that contains hashes that point to other files.
 
-Information passing through the reveal data channel does not have to be
-considered by layer 1: for this reason, the volume of information is
-not limited by the bandwidth of layer 1. Thus, the reveal data
-channel can be used to upload large volumes of data to the rollup.
+- A rollup node can request information about the rollup, including the address and origination level of the rollup, known as _metadata requests_.
 
-## Origination
+{/*
+TODO how is this data provided?
+Where does it come from?
+Do we need instructions on how to provide data?
+Eventually include:
+  - importing data from a DAC certificate (which can contain anything ultimately, including a kernel to upgrade to)
+  - revealing data from the (WIP) DAL
+*/}
+
+## Consensus
+
+Smart Rollup nodes achieve consensus differently from layer 1 nodes.
+Instead of constantly sharing blocks and updating their states, layer 2 nodes maintain their states internally and share them at predefined intervals.
+Because the nodes behave in a deterministic manner, their states should all be the same if they have processed the same Smart Rollup inbox messages.
+This state is referred to as the "state of the rollup."
+
+Rollups share and achieve consensus on the state of the rollup with these two intervals:
+
+- The _commitment period_ runs for 60 consecutive blocks.
+During the commitment period, the rollups receive messages from the Smart Rollup inbox, process them, and update their internal states.
+
+  At the end of the commitment period, Smart Rollup nodes use the PVM to generate a hash of their states and publish it to layer 1.
+  This hash is referred to as the node's _commitment_.
+  The commitment allows other nodes to verify that the node has responded to all of the messages during the commitment period and run the rollup code accurately.
+
+  Then a new commitment period begins.
+
+- The _refutation period_ for a commitment period starts when the commitment period ends.
+During the refutation period, if two or more commitments are different, the nodes play a _refutation game_ to identify the correct commitment.
+
+  Then the correct commitment is said to be _cemented_ and therefore final and unchangeable.
```

Comment:
> When there is only one commitment left, either because all nodes published identical commitments or the refutation game eliminated all incorrect commitments, this correct commitment is said to be _cemented_ and therefore final and unchangeable.